consul-lambda/go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/hashicorp/terraform-aws-consul-lambda/consul-lambda
 
-go 1.18
+go 1.19
```

Comment:
The go version in the [bin-ci.yaml](https://github.com/hashicorp/terraform-aws-consul-lambda/blob/9853030f0eab43ac410d59942413a23574a5691b/.github/workflows/bin-ci.yml#L6) is still set to `1.18`.. may as well bump CI to use `1.19`.